webgpu.yml

```diff
@@ -434,7 +499,8 @@ enums:
           "Core" profile which can be supported on Vulkan/Metal/D3D12 (at least).
   - name: feature_name
     doc: |
-      TODO
+      @deprecated alias for @ref WGPUDeviceFeatureName.
+      New feature names will NOT be added here.
```

Comment:
Best I could come up with for how to keep the aliases. (I could modify the generator so it can make enums alias other enums, but then there's an ordering dependency too, so this is much easier.)